src/Downloader.ts

```diff
@@ -586,19 +591,34 @@ class Downloader {
   }
 
   private async getCompressedBody(resp: any): Promise<any> {
-    return this.isMimeTypeImage(resp.headers['content-type']) ? await imagemin.buffer(resp.data, imageminOptions) : resp.data;
+    if (isBitmapImageMimeType(resp.headers['content-type'])) {
+      if (isWebpCandidateImage(this.webp, this.cssDependenceUrls.hasOwnProperty(resp.config.url), resp.headers['content-type'])) {
+        resp.data = await imagemin.buffer(resp.data, {
+          plugins: [
+            imageminWebp({ quality: 75 })
+          ]
+        });
```

Comment:
Instead of having one or the other imagemin options, you will probably have to create profiles. But all of this needs to be setup earlier.